rllib/utils/torch_utils.py

```diff
@@ -211,6 +211,9 @@ def explained_variance(y: TensorType, pred: TensorType) -> TensorType:
         The explained variance given a pair of labels and predictions.
     """
     y_var = torch.var(y, dim=[0])
+    if y_var == 0.0:
+        # Model case in which y does not vary with explained variance of -1
+        return torch.tensor(-1.0)
```

Comment:
We should do `torch.tensor(-1.0).to(pred.device)` instead to make sure this new code is also GPU compatible.